### PR TITLE
Add a cli option to adjust priority fees.

### PR DESCRIPTION
**Pull request by ankur2136**

### CODE REVIEW

**Review by Funkatronics — APPROVED** [review]

File: packages/cli/src/commands/create/CreateCliApp.ts [hunk 1]
```diff
@@ -70,6 +74,7 @@ export const createAppCommand = async ({
   dryRun,
   publisherMintAddress,
   storageParams,
+  priorityFeeLamports = 500000,
```
Comment (sdlaver) [inline packages/cli/src/commands/create/CreateCliApp.ts:77]: Can we pull this constant out somewhere, and import it where used? That'll make any future updates trivial.

File: packages/core/src/CoreUtils.ts [hunk 2]
```diff
@@ -24,10 +24,16 @@ type JsonMetadataMetaplexFile = Omit<JsonMetadata, "image"> & {
 export const mintNft = async (
   metaplex: Metaplex,
   json: JsonMetadataMetaplexFile,
-  createNftInput: Omit<CreateNftInput, "uri" | "name" | "sellerFeeBasisPoints">
+  createNftInput: Omit<CreateNftInput, "uri" | "name" | "sellerFeeBasisPoints">,
+  priorityFeeLamports: number = 500000,
 ): Promise<TransactionBuilder> => {
   console.info({ json });
   const { uri } = await metaplex.nfts().uploadMetadata(json);
+  const computeBudget = 250000
```
Comment (sdlaver) [inline packages/core/src/CoreUtils.ts:32]: Similar comment re: factoring out this constant.

**Review by sdlaver — APPROVED** [review]

File: packages/core/src/CoreUtils.ts (hunk 2 above)
Comment (ankur2136) [reply packages/core/src/CoreUtils.ts:32]: This one is dynamic and @Funkatronics was going to add simulation to get this value.

File: packages/cli/src/commands/create/CreateCliApp.ts (hunk 1 above)
Comment (ankur2136) [reply packages/cli/src/commands/create/CreateCliApp.ts]: Done.

**Review by sdlaver — APPROVED** [review]

**Pull request merged**